history-of-the-quakers-v4/original/02-book-6-chapter-15.adoc

```diff
@@ -1035,15 +1027,25 @@ that in the delightful fruition thereof, the Queen may enjoy many days,
 and after a life of comfort, be translated to a glorious immortality.
 Signed on behalf of the said people by
 
+[.signed-section-signature]
 John Field,
 
+[.signed-section-signature]
 Joseph Wyeth.
 
+--
+
+[.embedded-content-document]
```

Comment:
This is where I start forgetting and doubting what I should do with these little quotes.....maybe it just should have been a quote block?? AHHH!!!!